apps/webapp/src/domains/connectors/components/connection-card.tsx

```
@@ -295,6 +413,64 @@ export function ConnectionCard({
+										<label
+											key={repo.fullName}
+											className="flex items-center gap-3 text-fontSize-sm"
+										>
+											<Checkbox
+												checked={checked}
+												onCheckedChange={(value) =>
+													toggleRepositorySelection(repo.fullName, value === true)
+												}
+											/>
+											<span className="font-mono">{repo.fullName}</span>
+										</label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)